modules/nw-udn-support-matrix-primary-secondary.adoc

```diff
@@ -0,0 +1,324 @@
+
+:_mod-docs-content-type: CONCEPT
+[id="support-matrix-for-primary-secondary-networks_{context}"]
+= Support matrix for primary and secondary networks
+
+The `UserDefinedNetwork` and `NetworkAttachmentDefinition` custom resources (CRs) provide cluster administrators and users the ability to create customizable network configurations and define their own network topologies, ensure network isolation, manage IP addressing for workloads, and leverage advanced network features. A third CR, `ClusterUserDefinedNetwork`, is also available, which allows administrators the ability to create and define additional networks spanning multiple namespaces at the cluster level. Both user-defined networks and network attachment definitions can serve as both the primary and secondary network interface, and each support `layer2` and `layer3` topologies; a third network topology, Localnet, is also supported with network attachment definitions. 
+
+[NOTE]
+====
+As of {product-title} 4.18, the Localnet topology is unavailable for use with the `UserDefinedNetwork` and `ClusterUserDefinedNetwork` APIs. It is available for use with the `NetworkAttachmentDefinition` API only. 
+====
+
+The following section highlights the supported features of the `UserDefinedNetwork` and `NetworkAttachmentDefinition` CRs when they are used as either the primary or secondary network. A separate table for the `ClusterUserDefinedNetwork` CR is also included. 
+ 
+.Primary network support matrix
+[cols="1,1a,1a,1a,1a,1a, options="header"]
+|===
+| Network type | Network feature | Supported | Layer2 topology |Layer3 topology |Localnet topology ^[1]^
+
+
+.10+| Primary network (UDN and NAD)
+
+| *east-west traffic* (Miguel)
+^| &#10003;
+^| &#10003;
+^| &#10003;
+^| &#10003; (`net-attach-def` only)
+
+| *north-south traffic* (Miguel)
+^| &#10003;
+^| &#10003;
+^| &#10003;
+^| &#10003; (`net-attach-def` only)
+
+| *Persistent IPs* (Miguel)
+^| &#10003;
+^| &#10003;
+^| X
+^| &#10003; (`net-attach-def` only)
+
+| *Routes* (Ricky)
+^| &#10003;
+^| &#10003;
+^| &#10003;
+^| X
+
+| *Services* (Ricky)
+^| &#10003;
+^| &#10003;
+^| &#10003;
+^| X
+
+| *`EgressIP` resource* (martin)
+^| &#10003;
+^| &#10003;
+^| &#10003;
+^| X
+
+| *Multicast* ^[3]^ (Dumitru)
+^| &#10003;
+^| X
+^| &#10003;
+^| &#10003; (`net-attach-def` only)
+
+| *`MultinetworkPolicy` resource*
+^| X
+^| X
+^| X
+^| X
+
+| *`NetworkPolicy` resource* ^[2]^
+^| &#10003;
+^| X
+^| X
+^|
+
+|*Hybrid Overlay*
+^| X
+^| X
+^| X
+^|
+
+|*`AdminNetworkPolicy`* (ANP & BANP)
+^| &#10003;
+^| &#10003;
+^| &#10003;
+^|
+
+|===
+1. The Localnet topology is unavailable for use with the `UserDefinedNetwork` CR. 
+2. When creating a `UserDefinedNetwork` CR with a primary network type, network policies must be created _after_ the `UserDefinedNetwork` CR.
+3. Multicast must be enabled in the namespace, and it is only available between OVN-Kubernetes network pods. 
+
+.Secondary network support matrix
```

Comment:
You can have north south traffic with localnet on secondary networks.

Reply:
Fixed.